lib/generators/rails/seed_uuids/seed_uuids_generator.rb

```diff
@@ -1,3 +1,5 @@
+require 'rails/generators'
```

Comment:
I think this may have been existing behaviour (or possibly due to eager loading changes), running `db:seed` locally raised an error without this.